App/package.json

```diff
@@ -16,6 +16,7 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "axios": "^1.5.0",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
I don't think we need this dependency since the user events like functionalities can be obtained from `@testing-library/react`.  Just remove it and `npm i` again to reset the package-lock.json. Otherwise looks good.

Reply:
In the event we do actually need this library, it should be installed as a dev-dependency. To do this, after removing the line from the package.json file, instead of
`npm i @testing-library/user-event`, you would run `npm i @testing-library/user-event --save-dev`

Installing as a dev dependency essentially means means it'll be there when developing or running tests, but when we bundle the app to deploy it, the package won't be included in that bundle (meaning everything loads faster for the user). 

If you check the package.json file, you'll see we have other things already installed as dev dependencies, including Jest itself.

Reply:
I removed it, it was a leftover from a previous version of the test I had tried to make before asking Ghaith for help.